### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by fish-sushi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,129 @@
...
+
+```
+# Geth console
+> exit
+```
+
+```bash
+# Paste your account address to [account] part
+# Ex) geth --testnet --syncmode=full --rpc --shh --networkid=3 --unlock 0xceeb8b025e23deef03a9ead0ecb4a4be98282006 console
+geth --testnet --syncmode=full --rpc --shh --networkid=3 --unlock [account] console
```
Comment (c0smic0wl) [inline README.md]: `--unlock [index]` can also unlock the account. I think `--unlock 0` is easier.

**Review by c0smic0wl — APPROVED** [review]
:owl:

File: README.md (hunk 1 above)
Comment (parkkoon) [reply README.md]: full -> fast

File: README.md [hunk 2]
```diff
@@ -1,5 +1,129 @@
...
+## Before Start
+### Geth Client
+
+#### Sync blocks (Geth Client only)
+
+1. Open Terminal
+2. Paste and execute below code
+
+```bash
+geth --testnet --syncmode=fast --rpc --shh --networkid=3 console
```
Comment (parkkoon) [inline README.md]: Maybe we will not use ropsten. Replace 3 to 4 please.

**Review by parkkoon — CHANGES_REQUESTED** [review]

**Pull request merged**